s3/src/main/scala/akka/stream/alpakka/s3/model.scala

```diff
@@ -89,13 +99,13 @@ object MultipartUploadResult {
       location: Uri,
       bucket: String,
       key: String,
-      etag: String,
+      eTag: String,
```

Comment:
AFAIK this can break source compatibility (i.e. if someone calls `apply` with named parameters) but it doesn't break binary compatibility.